src/components/Register.js

```diff
@@ -1,10 +1,12 @@
 import React, {useState} from 'react';
+import {useNavigate} from "react-router";
+import "../styles/Form.css"
```

Comment:
Can you add a semi-colon at the end of the line so that its formatted correctly?